CONTRIBUTING.md

```diff
@@ -1,159 +1,243 @@
 
 1. Fork and clone the repository to a directory on your local machine.
+
 2. Open a command line in that directory and execute the following command:
+
     ```bash
+
     make newenv
+
     ```
-    Red, its dependencies, and all required development tools, are now installed to a virtual environment located in the `.venv` subdirectory. Red is installed in editable mode, meaning that edits you make to the source code in the repository will be reflected when you run Red.
+
+    Red, its dependencies, and all required development tools are now installed to a virtual environment located in the `.venv` subdirectory. Red is installed in editable mode, meaning that edits you make to the source code in the repository will be reflected when you run Red.
+
 3. Activate the new virtual environment with one of the following commands:
+
     - Posix:
+
         ```bash
+
         source .venv/bin/activate
+
         ```
+
     - Windows:
+
         ```powershell
+
         .venv\Scripts\activate
+
         ```
-    Each time you open a new command line, you should execute this command first. From here onwards, we will assume you are executing commands from within this activated virtual environment.
+
+    Each time you open a new command line, execute this command first. From here onwards, we will assume you are executing commands from within this activated virtual environment.
+
  
-**Note:** If you're comfortable with setting up virtual environments yourself and would rather do it manually, just run `pip install -Ur tools/dev-requirements.txt` after setting it up.
+
+**Note:** If you’re comfortable setting up virtual environments yourself and would rather do it manually, just run `pip install -Ur tools/dev-requirements.txt` after setting it up.
 
 ### 4.2 Testing
-We're using [tox](https://github.com/tox-dev/tox) to run all of our tests. It's extremely simple to use, and if you followed the previous section correctly, it is already installed to your virtual environment.
+
+We’re using [tox](https://github.com/tox-dev/tox) to run all of our tests. It’s extremely simple to use, and if you followed the previous section correctly, it is already installed in your virtual environment.
 
 Currently, tox does the following, creating its own virtual environments for each stage:
+
 - Runs all of our unit tests with [pytest](https://github.com/pytest-dev/pytest) on python 3.8 (test environment `py38`)
+
 - Ensures documentation builds without warnings, and all hyperlinks have a valid destination (test environment `docs`)
+
 - Ensures that the code meets our style guide with [black](https://github.com/psf/black) (test environment `style`)
 
-To run all of these tests, just run the command `tox` in the project directory.
+To run these tests, just run the command `tox` in the project directory.
```

Comment:
These two sentences have two different meanings, you should reinstall the meaning of the first which is to run all tests and not a subsection.